adam-core/src/main/scala/org/bdgenomics/adam/formats/avro/SingleReadBuckets.scala

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.formats.avro
```

Comment:
Yeah, I think I was experimenting with trying to import avdl's across projects, and seeing if this helped (it turned out to be that the avdl had to be in src/main/resources/avro in order to pull in the bdg-formats avdl non-locally).  I agree that this probably isn't the right package name (I'd probably prefer the old "models").